Source/AliveLib/Slig.cpp

```diff
@@ -6592,11 +6581,14 @@ BaseAliveGameObject* Slig::FindBeatTarget_4BD070(int /*a2*/, int gridBlocks)
             PSX_RECT bRect = {};
             pObj->vGetBoundingRect_424FD0(&bRect, 1);
 
-            if (pObj->field_10C_health > FP_FromInteger(0) &&
+            if (hitRect.w <= bRect.w &&
```

Comment:
why can this use PSX_Rects_overlap_no_adjustment?

Reply:
PSX_Rects_overlap_no_adjustment will return false if any of the params are exactly the same in both Rects, unlikely as it is.